### ISSUE TITLE
The tab order in the Combo dialog is invalid.

### ISSUE THREAD

**Issue by xmichelo** [xmichelo/Beeftext#57]
The tab order in the combo dialog is invalid: the group combo should be in second.

**Comment by alanfeng0** [comment]
The tab order is still invalid in the combo dialog

**Comment by xmichelo** [comment]
Hi. Thank you for reporting. For some reason, the snippet editor control is last in the the tab order list. I'll try to fix that ASAP.
